dotcom-rendering/src/web/components/Card/components/HeadlineWrapper.tsx

```diff
@@ -1,17 +1,46 @@
 import { css } from '@emotion/react';
+import { from } from '@guardian/source-foundations';
 
 type Props = {
 	children: React.ReactNode;
+	imagePosition?: ImagePositionType;
+	imagePositionOnMobile?: ImagePositionType;
+	imageUrl?: string;
+	starRating?: number;
 };
 
-export const HeadlineWrapper = ({ children }: Props) => (
+export const HeadlineWrapper = ({
+	children,
+	imagePosition,
+	imagePositionOnMobile,
+	imageUrl,
+	starRating,
```

Comment:
Passing these props here to make sure the top-padding change in this component will not affect all the fronts cards.